executor/src/witgen/data_structures/finalizable_data.rs

```diff
@@ -75,11 +76,69 @@ impl<T: FieldElement> CompactData<T> {
         }
     }
 
-    fn get(&self, row: usize, col: u64) -> (T, bool) {
+    pub fn append_new_rows(&mut self, count: usize) {
+        self.data
+            .resize(self.data.len() + count * self.column_count, T::zero());
+        self.known_cells
+            .grow(self.known_cells.len() + count * self.column_count, false);
```

Comment:
```suggestion
            .grow(count * self.column_count, false);
```
I think it should be this!